.github/workflows/auto-fix-lint.yml

```diff
@@ -14,19 +14,10 @@ jobs:
     if: github.actor != 'dependabot[bot]' && !github.event.pull_request.head.repo.fork
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - uses: sap/ai-sdk-js/.github/actions/setup@scripts
         with:
-          fetch-depth: 0
```

Comment:
this is now lost, I guess it might have been here for performance reasons? Not sure if it's worth it though, our repo is not that huge ;)

Reply:
I removed that, but didn't leave a comment. This was unnecessary and actually has a bad impact on performance because it fetches all history and all tags. (Kind of counter intuitive if you ask me.)